File: Programming-languages-and-laboratory/Java/Exercises/07_compito_java_2016_01_29/Azienda.java
import java.util.*;

public class Azienda extends Cliente {
    private String ragioneSociale;
    private int fatturato;

    public Azienda(int codice, String indirizzo, int giorno, int mese, int anno, String ragioneSociale, int fatturato) {
        super(codice, indirizzo, giorno, mese, anno);
        this.ragioneSociale = ragioneSociale;
        this.fatturato = fatturato;
    }

    public String toString() {
        return "azienda\t" + codice + "\t-\t" + ragioneSociale + "\t" + indirizzo + "\t" + giorno + "-" + mese + "-" + anno + "\t" + fatturato;
    }
}

File: Programming-languages-and-laboratory/Python/Exercises/07_compito_java_20160129/Azienda.py
from Cliente import Cliente


class Azienda(Cliente):
    def __init__(self, tipo, codice, indirizzo, giorno, mese, anno, ragione_sociale, fatturato):
        super().__init__(tipo, codice, indirizzo, giorno, mese, anno)
        self._ragione_sociale = ragione_sociale
        self._fatturato = fatturato
        
    def get_nome(self):
        return self._ragione_sociale

    def __str__(self):
        return "Azienda\t" + super().__str__() + "\t-\t" + self._ragione_sociale + "\t" + self._indirizzo + "\t" + str(self._giorno) + "-" + str(self._mese) + "-" + str(self._anno) + "\t" + str(self._fatturato)
